ticdc: update ticdc time-zone docs (#13850)
pingcap/docs: ticdc/ticdc-sink-to-mysql.md

| `ssl-ca` | The path of the CA certificate file needed to connect to the downstream MySQL instance (optional).  |
| `ssl-cert` | The path of the certificate file needed to connect to the downstream MySQL instance (optional). |
| `ssl-key` | The path of the certificate key file needed to connect to the downstream MySQL instance (optional). |

> | `time-zone` | The time zone used when connecting to the downstream MySQL instance, which is effective since v4.0.8. This is an optional parameter. If this parameter is not specified, the time zone of TiCDC service processes is used. If this parameter is set to an empty value, for example, `time-zone=""`, no time zone is specified when TiCDC connects to the downstream MySQL instance and the default time zone of the downstream is used. |

Reviewer: ```suggestion
| `time-zone` | The time zone used when connecting to the downstream MySQL instance, which is effective since v4.0.8. This is an optional parameter. If this parameter is not specified, the time zone of TiCDC service processes is used. If this parameter is set to an empty value, such as `time-zone=""`, no time zone is specified when TiCDC connects to the downstream MySQL instance and the default time zone of the downstream is used. |
```
